src/CoreWCF.Primitives/src/CoreWCF/Description/DispatcherBuilder.cs

```diff
@@ -590,6 +593,16 @@ private static void BuildDispatchOperation(OperationDescription operation, Dispa
 
                 parent.UnhandledDispatchOperation = child;
             }
+
+            var authorizationPolicyProvider = services.GetService<IAuthorizationPolicyProvider>();
```

Comment:
We ended up adding some perf tweaks to enable caching when the IAuthorizationPolicyProvider is the Default implementation, because we know this always returns the same policies, which allows caching of the combined policies.  I'm not sure if this code is called more than once, but food for thought.  For 7.0 we ended up sticking the combined policy into the endpoint metadata itself. PR for context: https://github.com/dotnet/aspnetcore/pull/43124

Reply:
This code should only be called once at application startup, the combined policy is stored in the `DispatchOperation`.